docs/docs/started/troubleshooting.md

```diff
@@ -0,0 +1,5 @@
+---
+id: troubleshooting
+---
+
+# Troubleshooting
```

Comment:
This page should be deleted, right?

Reply:
yes

Reply:
We do have an issue for this correct? - just making sure this will get done... eventually